src/compiler/compile/render_dom/wrappers/Element/index.ts

```diff
@@ -304,45 +302,68 @@ export default class ElementWrapper extends Wrapper {
 			if (${this.var}) ${this.var}.m(${parent_node || '#target'}, ${parent_node ? 'null' : '#anchor'});
 		`);
 
-		const anchor = this.get_or_create_anchor(block, parent_node, parent_nodes);
-		const has_transitions = !!(this.node.intro || this.node.outro);
-		const not_equal = this.renderer.component.component_options.immutable ? x`@not_equal` : x`@safe_not_equal`;
-
-		block.chunks.update.push(b`
-			if (${tag}) {
-				if (!${previous_tag}) {
-					${this.var} = ${this.child_dynamic_element_block.name}(#ctx);
-					${this.var}.c();
-					${has_transitions && b`@transition_in(${this.var})`}
-					${this.var}.m(${this.get_update_mount_node(anchor)}, ${anchor});
-				} else if (${not_equal}(${previous_tag}, ${tag})) {
-					${this.var}.d(1);
-					${this.renderer.options.dev && b`@validate_dynamic_element(${tag});`}
-					${this.renderer.options.dev && this.node.children.length > 0 && b`@validate_void_dynamic_element(${tag});`}
-					${this.var} = ${this.child_dynamic_element_block.name}(#ctx);
-					${this.var}.c();
-					${this.var}.m(${this.get_update_mount_node(anchor)}, ${anchor});
-				} else {
-					${this.var}.p(#ctx, #dirty);
-				}
-			} else if (${previous_tag}) {
-				${
-					has_transitions
-						? b`
-							@group_outros();
-							@transition_out(${this.var}, 1, 1, () => {
+		if (this.node.tag_expr.dynamic_dependencies().length == 0) {
+			block.chunks.update.push(b`if (${this.var}) ${this.var}.p(#ctx, #dirty)`);
```

Comment:
I think it's better to handle this in another PR because we can optimize other functions also.
(I already implemented this in the past and can issue a PR right away)

Reply:
So, should I remove these lines ?

(sorry I'm not used to these processes)

Reply:
We merged this optimization in https://github.com/sveltejs/svelte/pull/7938.
So could you please revert this change?